plugin-fancymenu/lxqtfancymenuappmap.cpp

```diff
@@ -208,24 +208,26 @@ QVector<const LXQtFancyMenuAppMap::AppItem *> LXQtFancyMenuAppMap::getMatchingAp
     QVector<const AppItem *> byKeyword;
 
     //TODO: implement some kind of score to get better matches on top
+    QString queryLower = query.toLower();
 
     for(const AppItem *app : qAsConst(mAppSortedByName))
     {
-        if(app->title.contains(query))
+        if(app->title.toLower().contains(queryLower))
         {
             byName.append(app);
             continue;
         }
 
-        if(app->comment.contains(query))
+        if(app->comment.toLower().contains(queryLower))
         {
             byKeyword.append(app);
             continue;
         }
 
+        // Keywords are already converted to lower case
         for(const QString& key : app->keywords)
         {
-            if(key.startsWith(query))
+            if(key.startsWith(queryLower))
```

Comment:
Please use `Qt::CaseInsensitive` instead of `toLower()`.

Reply:
Since keywords are already stored lowercase ins't it more efficient to convert to lower only once at beginning of search instead of doing case insensitive match for each item?

Reply:
We can't be sure of the efficiency of micro-improvements, while they damage code readability with no tangible effect. Qt already has `Qt::CaseInsensitive` for insensitive search.

Reply:
So should I drop `toLower()` inside item creation too?

Reply:
Yes; if it isn't needed anywhere else, it's a redundant computation.